cmd/expiration-mailer/main.go

```diff
@@ -267,16 +268,18 @@ func (m *mailer) findExpiringCertificates() error {
 			)
 		} else {
 			_, err = m.dbMap.Select(
-				&certs,
-				`SELECT cert.* FROM certificates AS cert
-						 JOIN certificateStatus AS cs
-						 ON cs.serial = cert.serial
-						 AND cert.expires > :cutoffA
-						 AND cert.expires <= :cutoffB
-						 AND cs.status != "revoked"
-						 AND COALESCE(TIMESTAMPDIFF(SECOND, cs.lastExpirationNagSent, cert.expires) > :nagCutoff, 1)
-						 ORDER BY cert.expires ASC
-						 LIMIT :limit`,
+				&statuses,
+				`SELECT
```

Comment:
Reformatting this section makes it harder to spot the diff. Can you leave the formatting the same? We're going to delete this section anyhow so it's less important to reformat.